SDL/SDLJoystick.cpp

```diff
@@ -41,16 +41,19 @@ SDLJoystick::SDLJoystick(bool init_SDL ) : registeredAsEventHandler(false) {
 		cout << "gamecontrollerdb.txt missing" << endl;
 	}
 	cout << "SUCCESS!" << endl;
-	setUpControllers();
+	//setUpControllers();
```

Comment:
It seems like this comments out all calls to `setUpControllers()`, but also modifies it.  That's confusing.

Git remembers code forever, so if we don't need this anymore the line should be removed and so should the entire method.

-[Unknown]